src/libraries/System.Drawing.Common/src/CompatibilitySuppressions.xml

```diff
@@ -30,6 +30,150 @@
     <Left>lib/netstandard2.0/System.Drawing.Common.dll</Left>
     <Right>lib/net462/System.Drawing.Common.dll</Right>
   </Suppression>
+  <Suppression>
```

Comment:
Can you please add a comment above this element to indicate that these type forwards exist for back-compatibility with xamarin frameworks?